src/Signature/SignatureV4.php

```diff
@@ -67,19 +67,22 @@ public function signRequest(
     public function presign(
         RequestInterface $request,
         CredentialsInterface $credentials,
-        $expires
+        $expires,
+        $startsAt = null
```

Comment:
Instead of an optional positional argument, you could use an optional array of named arguments:
```php
public function presign(<args>, array $options = [])
{
     $startTimestamp = isset($options['start_time']) ? $options['start_time'] : time();
```